lua/warden/convars.lua

```diff
@@ -0,0 +1,8 @@
+if SERVER then
+	CreateConVar("warden_freeze_disconnect", 1, nil, "Freeze owned entities on player disconnect", 0, 1)
+	CreateConVar("warden_cleanup_disconnect", 1, nil, "Cleanup owned entities on player disconnect", 0, 1)
+	CreateConVar("warden_cleanup_time", 600, nil, "Time in seconds until cleanup after player disconnect", 0)
```

Comment:
These need to have FCVAR_ARCHIVE as the third argument

Reply:
FCVAR_ARCHIVE is client only
![image](https://user-images.githubusercontent.com/59924045/224857512-2c969447-92bb-4f53-a41c-077e35d95bfb.png)